test/src/org/exist/xmldb/XmldbURITest.java

```diff
@@ -23,24 +21,27 @@ public void xmldbURIConstructors() throws URISyntaxException {
         XmldbURI.xmldbUriFor("xmldb:exist:///db");
         XmldbURI.xmldbUriFor("xmldb:exist://localhost/db");
 
-        XmldbURI.xmldbUriFor("xmldb:exist://localhost:" + jettyPort + "/db");
-        XmldbURI.xmldbUriFor("//localhost:" + jettyPort + "/db");
+        XmldbURI.xmldbUriFor("xmldb:exist://localhost:8080/db");
```

Comment:
I am quite concerned ... this 'flexible' port thing was introduced by @ljo to be able to run tests in parallel on a system.  Or run a test in parallel to an exist-db instance running on the same server. This is not possible anymore.....  Probably introduced with an other PR but now we are drifting away... @adamretter @wolfgangmm  ?